src/channel.ts

```diff
@@ -4,8 +4,7 @@ import { Filter } from 'nostr-tools';
 import { NostrPool, NostrEvent } from '.';
 
 export async function listChannels(pool: NostrPool): Promise<ChannelInfo[]> {
-  // todo: this should only use the store, not go and re-query stuff, being lazy to get things done
-  return (await pool.list([{ kinds: [40] }])).map((ent) => {
+  return (await pool.list([{ kinds: [40] }], true)).map((ent) => {
```

Comment:
db_only should be a parameter passed to listChannels i think, the same way the rest are.   at least until we're sure the db is "good".  its nice to be able to turn it off and check things.

Reply:
I've updated it